renderer-macos/lib/stubs/BriskTextInput.c

```diff
@@ -0,0 +1,15 @@
+@interface BriskTextInput : NSTextInputContext <NSTextInputClient>
+
+@end
+
+@implementation BriskTextInput
+
+@end
+
+NSTextInputContext *ml_BriskTextInput_make() {
+  BriskTextInput *input = [BriskTextInput new];
```

Comment:
so `ml_BriskTextInput_make` should return a view (Like `NSTextField` or editable `NSTextView`.) `NSTextInputContext` is just a protocol (think java interface, functor signature (kinda)) related to text input.

Reply:
So [NSTextInputContext](https://developer.apple.com/documentation/appkit/nstextinputcontext?language=objc) is actually a class and [NSTextInputClient](https://developer.apple.com/documentation/appkit/nstextinputclient?language=objc) is a protocol. Should `ml_BriskTextInput_make` then return a `NSTextField` or `NSTextView` conforming to `NSTextInputClient` protocol?